crypto/evp/ctrl_params_translate.c

```diff
@@ -2093,6 +2093,14 @@ static const struct translation_st evp_pkey_ctx_translations[] = {
       EVP_PKEY_CTRL_RSA_KEYGEN_PRIMES, "rsa_keygen_primes", NULL,
       OSSL_PKEY_PARAM_RSA_PRIMES, OSSL_PARAM_UNSIGNED_INTEGER, NULL },
 
+    /*-
+     * SipHash
+     * ======
+     */
+    { SET, -1, -1, -1,
```

Comment:
Looks ok to me - maybe @levitte needs to confirm this one//

Reply:
The third -1 should probably be `EVP_PKEY_OP_TYPE_SIG`...  remember that this is for the old way of making MACs, via the PKEY signature API

Reply:
I wasn't sure what was appropriate there and _-1_ worked :)

Reply:
Fix submitted.